File: ncarg2d/man/threed/frst3.m
.\"
.\"	$Id: frst3.m,v 1.1 1993-03-11 16:34:33 haley Exp $
.\"
.TH FRST3 3NCARG "March 1993" UNIX "NCAR GRAPHICS"
.SH NAME
FRST3 - Positions the pen on a point in 3-space
.SH SYNOPSIS
CALL FRST3 (UA,VA,WA)
.SH C-BINDING SYNOPSIS
#include <ncarg/ncargC.h>
.sp
void c_FRST3 (float ua, float va, float wa)
.SH DESCRIPTION 
.IP "UA, VA, WA" 12
coordinates for the point in 3-space.
.SH C-BINDING DESCRIPTION
The C-binding argument descriptions are the same as the Fortran 
argument descriptions.
.SH ACCESS
To use FRST3 load the NCAR Graphics libraries ncarg, ncarg_gks,
and ncarg_loc, preferably in that order.  To use c_frst3 load 
the NCAR Graphics libraries ncargC, ncarg_gksC, ncarg, ncarg_gks,
and ncarg_loc, preferably in that order.
.sp
.SH SEE ALSO
Online:
curve3 fence3 frst3 line3 perim3 point3 set3 threed
tick43 vect3 ncarg_cbind
.sp
Hardcopy:  "NCAR Graphics User's Guide, Version 2.00"
.SH COPYRIGHT
(c) Copyright 1987, 1988, 1989, 1991, 1993 University Corporation
for Atmospheric Research
.br
All Rights Reserved

File: ncarg2d/src/libncarg/threed/frst3.f
C
C $Id: frst3.f,v 1.6 2008-07-27 00:17:33 haley Exp $
C
C                Copyright (C)  2000
C        University Corporation for Atmospheric Research
C                All Rights Reserved
C
C The use of this Software is governed by a License Agreement.
C
      SUBROUTINE FRST3 (U,V,W)
      XDUM = 5.
      CALL TRN32T (U,V,W,X,Y,XDUM,2)
      CALL PLOTIT  (32*INT(X),32*INT(Y),0)
      RETURN
      END
